firebase-installations/src/main/java/com/google/firebase/installations/GetIdListener.java

```diff
@@ -42,8 +42,7 @@ public boolean onStateReached(PersistedInstallationEntry persistedInstallationEn
   }
 
   @Override
-  public boolean onException(
-      PersistedInstallationEntry persistedInstallationEntry, Exception exception) {
+  public boolean onException(Exception exception) {
     return false;
```

Comment:
why don't we fail the completion source if an exception is triggered in the getId path?

Reply:
getId flow should never trigger to propagate exception because we immediately return an FID (when available or generating a new one) even before trying to register the FID with FIS servers.